include/boost/gil/histogram.hpp

```diff
@@ -428,11 +440,12 @@ histogram<T...> cumulative_histogram(histogram<T...>& hist)
         "Cumulative histogram not possible of this type");
     
     using histogram_t = histogram<T...>;
-    using pair_t = std::pair<typename histogram_t::key_type, typename histogram_t::mapped_type>;
+    using pair_t  = std::pair<typename histogram_t::key_type, typename histogram_t::mapped_type>;
```

Comment:
nitpicking: remove the additional space before `=`

Reply:
I think it got added when I used clang-format, maybe to keep the equal signs in line 473 and 474 at the same position. This has been done in many other places also I think.

Reply:
@codejaeger Which clang-format configuration file did you use?

Reply:
@mloskot I believe I had used the one linked below:
[https://github.com/boostorg/gil/blob/develop/example/clang-format/.clang-format](https://github.com/boostorg/gil/blob/develop/example/clang-format/.clang-format)